relay/pkg/block/forwarder.go

```diff
@@ -27,6 +27,10 @@ const (
 	// Duration for which the forwarder should rest after performing
 	// a push action.
 	forwarderPushingSleepTime = 45 * time.Second
+
+	// Default duration for which the forwarder should rest after reaching the tip of
+	// Bitcoin blockchain
+	defaultForwarderPullingSleepTime = 60 * time.Second
```

Comment:
Why do we name it `default*` given that it is not configurable and the rest of the constants defined in this section are not prefixed with `default`?

Reply:
`forwarderPullingSleepTime` inside `Forwarder` is configurable - in tests we use a shorter value, while in real implementation we use `defaultForwarderPullingSleepTime`.

Perhaps we could name this constant `forwarderPullingSleepTime`, while the field inside `Forwarder` could be named `poolingSleepTime`?

Reply:
Maybe `pullingSleepTime`?

Reply:
Yes :)

Reply:
I see. Do we plan to make this value configurable in the config file? Because if so, prefix makes sense. Otherwise, I'd name it `forwarderPullingSleepTime` in both cases, we can use the same name.

Reply:
We do not plan to make it configurable via config file at this moment.
I'm ok with `forwarderPullingSleepTime` both in Forwarder and in const section.

Reply:
`pullingSleepTime` is even better!